scripts/post-changelog/post-changelog.ts

```diff
@@ -202,12 +282,16 @@ async function configureWebhookURL() {
  */
 async function publishChangelog(npmPackage: string) {
   try {
-    await simpleGit().fetch(['origin', 'main', '--tags']);
-    const tags = await simpleGit().tags({ '--sort': '-taggerdate' });
-    const diff = await getDiff(
-      argv.dry ? '@lmc-eu/spirit-web-react@3.1.0' : (tags.latest ?? ''),
-      changelogPath(npmPackage),
-    );
+    const diff = await executeGitOperationWithRetry(async () => {
+      await simpleGit().fetch(['origin', 'main', '--tags']);
+      const tags = await simpleGit().tags({ '--sort': '-taggerdate' });
+      const diff = await getDiff(
+        argv.dry ? '@lmc-eu/spirit-web-react@3.1.0' : (tags.latest ?? ''),
```

Comment:
isn't this a problem, to have hardcoded version here?

Reply:
I think no. You just want to print to output the changelog of some version, so I have picked one.